### DISCUSSION TITLE
Contribution Example and Details on the Front Page?

### DISCUSSION THREAD

**Discussion by VerzatileDev** [VerzatileDevOrg/Programming_HandBook#73] [General]
The question here is that if the Contribution guideline for the " Open Source " book should be done by or on the website. 
Or should it be written on the Wiki in detail from the Setup all the way to what pages require, what commands can be called and etc?

**Comment by JDSherbert** [comment]
Up to you! Take a look at some other repos and see what they do if unsure.

**Reply by VerzatileDev** [reply]
I do think that not a lot of people know that Github even has such a thing as a Wiki, which in itself is a problem as no one generally aims to use it. Though it is possible to somehow link the Wiki to the website, but I'm not 100% sure how to yet. Then again if using the front page for it would benefit to make it more accessible and changeable for accessibility and other causes. 

Generally most repositories would do it in a minimal summary of the Read me and then split it up with " Read it yourself and figure it out " type of style, which may cause issues that most repositories has is hundreds of Pull-requests, Wrong issues and a list of other problems which would require moderation and setup of a ruleset I would imagine.

**Comment by VerzatileDev** [comment]
Currently the contribution guidelines for Programmign Handbook are implemented in https://github.com/VerzatileDevOrg/Programming_HandBook/wiki 

With the following closes the dicsussion
